### ISSUE TITLE
Add s390x and ppc64le to binaries to releases

### ISSUE THREAD

**Issue by james-crowley** [git-lfs/git-lfs#3982]
**Describe the issue**
@bk2204 Currently only `arm64`, `amd64` and `i386` are being put on the releases for git-lfs. This creates an issue when going to use other architectures like `s390x` and `ppc64le`, since we have to build from source.

It would be great to have `s390x` and `ppc64le` binaries added to releases for convenience.
 
Building `s390x` binary using the build from source directions work fine, see below for confirmation. 

I could not tell how you are producing your builds. I saw multiple CI/CD, travis and circle. You be amazing if releases could get up in Package Cloud but I am not familiar with them to know if they support `s390x` and `ppc64le`. 

If you need access to `s390x` and `ppc64le` please let me know. There are multiple ways to give you access. 

**System environment**
```
root@eda37e058ace:/go/bin# ./git-lfs
git-lfs/2.9.0 (GitHub; linux s390x; go 1.13.6)
Sorry, no usage text found for "git-lfs"
root@eda37e058ace:/go/bin# ./git-lfs install
Git LFS initialized.
root@eda37e058ace:/go/bin# uname -m
s390x
```

**Comment by bk2204** [comment]
Hey,

Thanks for writing in.  The CI system we're using is GitHub Actions, and the workflows are in `.github/workflows`.  The others are obsolete.

We can certainly build binaries for `ppc64le` and `s390x`; that shouldn't be a problem, since Go will cross-compile with no extra steps.  I'm happy to take care of that.

Building for packagecloud is different, since that requires that a Docker container be set up and the `amd64` system will need to be set up with qemu cross-compiling.  If you want that part, I can point you to some stalled work in git-lfs/build-dockers#28 and #3846 that is designed around building `arm` and `arm64`, which you're welcome to leverage to build additional architectures.  Debian already ships with suitable versions of qemu to build, so it shouldn't be too difficult to make that work.  However, I don't plan to pick that work up myself, since I have no way of testing it.

If you want to add support for this, I'd recommend creating a separate CI job (and hence a separate mode of the script) to build the non-x86 architectures, since they'll likely take a bunch longer than the native ones and it would be better to have them run in parallel.

**Comment by james-crowley** [comment]
@bk2204 Thanks for already starting the PR to add binaries for `s390x` and `ppc64le`!

Does packagecloud only support running on a `amd64` system? The Jenkins team I am working with uses Qemu to do some cross compiling. I have not tried it myself, as I utilize actually machines. I know Travis now has support for `s390x`, `ppc64le`, `arm64`, etc. But as you mention you are now bought into GitHub Actions. 

> If you want to add support for this, I'd recommend creating a separate CI job 
Do you mean within GitHub Actions or in another CI/CD pipeline like Travis?

As for the binaries, are you going to push them out to the currently release? That would be extremely helpful!

**Comment by bk2204** [comment]
packagecloud doesn't build binaries; it just stores the packages we upload.  We build them as part of the CI and release workflows on Actions.  As far as I know packagecloud supports any architecture that we can provide it.  Other projects appear to have uploaded binaries for `arm64` and `mips`, at least.

I'd recommend using the qemu Linux `binfmt_misc` to run Docker containers on Actions for CI.  We've intentionally standardized on one CI provider and wouldn't want to add Travis back in.

As for the binaries, I plan to do a release this month (2.10), so those binaries will be included in that release.

**Comment by james-crowley** [comment]
@bk2204 If I get time to take a look at Qemu, I'll try to help on the PR/issues you mentioned.

Thanks for building and producing for `s390x` and `ppc64le`. Should be easy enough to pull the binary down via curl and install it. 

Ideally, I will get sometime to help out and we can get the releases up in packagecloud!

**Comment by james-crowley** [comment]
Closing out for now. Will open a new issue if work is done on packagecloud.
